tools/harness-automation/README.rst

```diff
@@ -0,0 +1,64 @@
+=======================
+Harness Automation Tool
+=======================
+
+This is a tool to automate testing openthread with GRL thread harness.
```

Comment:
Indicate the exact version of the Thread harness that this tool was tested against.